invenio_communities/communities/records/jsonschemas/communities/communities-v1.0.0.json

```diff
@@ -35,8 +35,8 @@
             "closed"
           ]
         },
-        "record_policy": {
-          "description": "Community record policy.",
+        "record_submission_policy": {
```

Comment:
Major: If we rename here, this requires data migration and a major bump.
@slint : This is unnecessary. The current name is good enough.